### PR TITLE
Enlarge your DG with one simple trick

### PR DESCRIPTION
**Pull request by hmenke**
:joy: :ok_hand: :100:

### CODE REVIEW

File: doc/sphinx/dg.rst [hunk 1]
```diff
@@ -1,3 +1,4 @@
+=================
```
Comment (KaiSzuttor) [inline doc/sphinx/dg.rst:1]: not needed, see http://docutils.sourceforge.net/docs/user/rst/quickref.html#section-structure

File: doc/sphinx/dg.rst [hunk 2]
```diff
@@ -195,7 +196,7 @@ developers.
 Source code structure
 ---------------------
 The source tree has the following structure:
-* src: The actual source code
+  * src: The actual source code
```
Comment (KaiSzuttor) [inline doc/sphinx/dg.rst]: that indentation was on purpose:
src
  core
  python/espressomd
.
.
.

**Review by KaiSzuttor — CHANGES_REQUESTED** [review]

File: doc/sphinx/dg.rst (hunk 2 above)
Comment (hmenke) [reply doc/sphinx/dg.rst]: I see.  Sphinx was complaining.

File: doc/sphinx/dg.rst (hunk 1 above)
Comment (hmenke) [reply doc/sphinx/dg.rst:1]: Totally needed.  Compare the screenshot to http://espressomd.org/html/doc/index.html

screenshot from 2017-08-11 22-45-04

Comment (KaiSzuttor) [reply doc/sphinx/dg.rst:1]: The current online documentation does not yet correspond to the current pyhton branch. I havent checked if the overline changes sth but if so the doc of sphinx is wrong.

**Pull request closed without merging**